Innowise.Clinic.Auth.Extensions/AuthTokenPairExtensions.cs

```diff
@@ -0,0 +1,31 @@
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using Innowise.Clinic.Auth.Dto;
+using Innowise.Clinic.Auth.Jwt.Exceptions;
+
+namespace Innowise.Clinic.Auth.Extensions;
+
+public static class AuthTokenPairExtensions
+{
+    public static Guid GetRefreshTokenId(this AuthTokenPairDto tokens)
+    {
+        var refreshToken = new JwtSecurityToken(tokens.RefreshToken);
+
+        var tokenIdClaim = refreshToken.Claims.SingleOrDefault(x => x.Type == "jti");
```

Comment:
The database is not used here. The method extracts claim from the refresh token. Valid refresh token can only have one Id claim. So I believe the exception should be thrown if token has more than one claim. It would be a signal that the token is invalid.